poorbook-front/src/components/NotesBox.js

```diff
@@ -1,69 +1,39 @@
-import React, { useState, useEffect, useRef } from 'react';
-import axios from 'axios';
+import React, { useState } from 'react';
+import axiosInstance from '../axiosInstance';
 
 const NotesBox = () => {
   const [notes, setNotes] = useState([]);
-  const [currentNote, setCurrentNote] = useState('');
-  const inputRef = useRef(null);
 
-  useEffect(() => {
-    const handleClickOutside = (event) => {
-      if (inputRef.current && !inputRef.current.contains(event.target)) {
-        saveCurrentNote();
+  const handlePasteNote = async () => {
+    try {
+      const text = await navigator.clipboard.readText();
+      if (text) {
+        const noteData = { content: text };
```

Comment:
Naming in backend is like:
{
  "noteContent": "string",
}

Reply:
backend should be changed to 'content'